flink-connector-aws-base/src/main/java/org/apache/flink/connector/aws/util/AWSAsyncSinkUtil.java

```diff
@@ -161,4 +160,64 @@ S createAwsAsyncClient(
                 .region(getRegion(configProps))
                 .build();
     }
+
+    /**
+     * @param configProps configuration properties
+     * @param httpClient the underlying HTTP client used to talk to AWS
+     * @param clientBuilder httpClientBuilder to build the underlying HTTP client
+     * @param awsUserAgentPrefixFormat user agent prefix for Flink
+     * @param awsClientUserAgentPrefix user agent prefix for kinesis client
+     * @return a new AWS Sync Client
+     */
+    public static <
+                    S extends SdkClient,
+                    T extends
+                            AwsSyncClientBuilder<? extends T, S> & AwsClientBuilder<? extends T, S>>
+            S createAwsSyncClient(
```

Comment:
I think refactoring it to AWSGeneralUtil will be messy as some Client Creation logic will have to be refactored to AWSGeneralUtil. Can I suggest renaming `AWSAsyncSinkUtil` to `AWSClientUtil` instead? This will be more tidy as all the utils to create client are in this class.